documentation/modules/con-cluster-operator-rbac.adoc

```diff
@@ -27,11 +27,10 @@ When the Cluster Operator deploys resources for a desired `Kafka` resource it al
 * The Kafka broker pods use a `ServiceAccount` called `_cluster-name_-kafka`
   - When the rack feature is used, the `strimzi-_cluster-name_-kafka-init` `ClusterRoleBinding` is used to grant this `ServiceAccount` access to the nodes within the cluster via a `ClusterRole` called `strimzi-kafka-broker`
   - When the rack feature is not used no binding is created.
-* The Zookeeper pods use the default `ServiceAccount`, as they do not need access to the Kubernetes resources.
-* The Topic Operator pod uses a `ServiceAccount` called `_cluster-name_-topic-operator`
-    - The Topic Operator produces Kubernetes events with status information, so the `ServiceAccount` is bound to a `ClusterRole` called `strimzi-topic-operator` which grants this access via the `strimzi-topic-operator-role-binding` `RoleBinding`.
-
-The pods for `KafkaConnect` and `KafkaConnectS2I` resources use the default `ServiceAccount`, as they do not require access to the Kubernetes resources.
+* The Zookeeper pods use a `ServiceAccount` called `_cluster-name_-zookeeper`
+* The Entity Operator pod uses a `ServiceAccount` called `_cluster-name_-entity-operator`
+    - The Topic Operator and User Operator produce Kubernetes events with status information, so the `ServiceAccount` is bound to a `ClusterRole` called `strimzi-entity-operator` which grants this access via the `strimzi-entity-operator` `RoleBinding`.
```

Comment:
I think it is only the topic-operator which produces the `Events`. But since the run as separate containers in the same pod the service account is used for both.